packages/client/src/LensClient.spec.ts

```diff
@@ -1,5 +1,6 @@
 import LensClient, { Profile, Reactions, Search, Explore } from '.';
 import { mumbaiSandbox } from './consts/environments';
+import { Revenue } from './revenue';
```

Comment:
this should be exported from '.' just to make sure it is also exported from the package